tests/installation/await_install.pm

```diff
@@ -13,7 +13,6 @@
 # - Check return code of check_screen against array @tags
```

Comment:
need to remove this, right? And there is a duplicated line (L16) if you dont mind

Reply:
L16 removed. This check_screen against @tags is still exist, so we should keep this.